fashion-messaging-app-api/server.js

```diff
@@ -16,6 +23,27 @@ app.use(cors({
 app.use(express.json()); 
 app.use(morgan('combined')); 
 
+const storage = multer.diskStorage({
+  destination: function(req,file,cb) {
+    cb(null, 'images/')
+  },
+  filename: function(req, file, cb) {
+    cb(null, Date.now() + path.extname(file.originalname));
+  },
+});
+
+const upload = multer({
+  storage: storage,
+  limits: { fileSize: '1000000'},
+  fileFilter: (req, file, cb) => {
+if (!file.originalname.match(/\.(jpg|jpeg|png|gif)$/i)) {
+  return cb(new Error('Only image files are allowed!'), false);
+  }
+  cb(null, true);
+},
+  limits: { fileSize: 1024 * 1024 * 10 }
+});
+app.use('/images', express.static(path.join(__dirname, 'images')));
```

Comment:
This is how I defined the muster functionality that is used to send the HTTP request for the file from the client to the server